etc/internal_options.conf

```diff
@@ -14,11 +14,11 @@
 # Analysisd default rule timeframe.
 analysisd.default_timeframe=360
 # Analysisd stats maximum diff.
-analysisd.stats_maxdiff=25000
+analysisd.stats_maxdiff=999000
```

Comment:
Oh yes, been using for a while. Logs can fluctuate in size quite a bit and the original defaults were way too low.